### ISSUE TITLE
snapshots/20210512

### ISSUE THREAD

**Issue by dvdhrm** [osbuild/rpmrepo#5]
This is a tracking issue for the upcoming snapshots scheduled for `20210512`.

**Comment by dvdhrm** [comment]
```
++ cat ./repo/f34-x86_64-updates-released.json
+ TARGET_JSON='{
"base-url": "https://dl01.fedoraproject.org/pub/fedora/linux/updates/34/Everything/x86_64/",
"platform-id": "f34",
"snapshot-id": "f34-x86_64-updates-released",
"storage": "public"
}'
[...]
[MIRROR] python-nashpy-doc-0.0.21-1.fc34.noarch.rpm: Status code: 404 for https://dl01.fedoraproject.org/pub/fedora/linux/updates/34/Everything/x86_64/Packages/p/python-nashpy-doc-0.0.21-1.fc34.noarch.rpm (IP: 38.145.60.22)
(3783/5501): python-mongoengine-doc-0.23.0-1.fc 3.5 MB/s \| 288 kB     00:00
[MIRROR] python-nashpy-doc-0.0.21-1.fc34.noarch.rpm: Status code: 404 for https://dl01.fedoraproject.org/pub/fedora/linux/updates/34/Everything/x86_64/Packages/p/python-nashpy-doc-0.0.21-1.fc34.noarch.rpm (IP: 38.145.60.22)
(3784/5501): python-neovim-doc-0.4.3-1.fc34.noa 2.1 MB/s \| 176 kB     00:00
[MIRROR] python-nashpy-doc-0.0.21-1.fc34.noarch.rpm: Status code: 404 for https://dl01.fedoraproject.org/pub/fedora/linux/updates/34/Everything/x86_64/Packages/p/python-nashpy-doc-0.0.21-1.fc34.noarch.rpm (IP: 38.145.60.22)
[MIRROR] python-nashpy-doc-0.0.21-1.fc34.noarch.rpm: Status code: 404 for https://dl01.fedoraproject.org/pub/fedora/linux/updates/34/Everything/x86_64/Packages/p/python-nashpy-doc-0.0.21-1.fc34.noarch.rpm (IP: 38.145.60.22)
[FAILED] python-nashpy-doc-0.0.21-1.fc34.noarch.rpm: No more mirrors to try - All mirrors were already tried without success
```

**Comment by dvdhrm** [comment]
I rescheduled `f34-x86_64-updates-released`. Now all 71 jobs succeeded.

```sh
curl https://rpmrepo.osbuild.org/v2/enumerate | jq '[.[] | select(test("20210512"))]'
[
  "cs8-aarch64-appstream-20210512",
  "cs8-aarch64-baseos-20210512",
  "cs8-ppc64le-appstream-20210512",
  "cs8-ppc64le-baseos-20210512",
  "cs8-x86_64-appstream-20210512",
  "cs8-x86_64-baseos-20210512",
  "el8-aarch64-appstream-n8.4-20210512",
  "el8-aarch64-appstream-n8.5-20210512",
  "el8-aarch64-appstream-r8.2-20210512",
  "el8-aarch64-appstream-r8.3-20210512",
  "el8-aarch64-baseos-n8.4-20210512",
  "el8-aarch64-baseos-n8.5-20210512",
  "el8-aarch64-baseos-r8.2-20210512",
  "el8-aarch64-baseos-r8.3-20210512",
  "el8-ppc64le-appstream-n8.4-20210512",
  "el8-ppc64le-appstream-n8.5-20210512",
  "el8-ppc64le-appstream-r8.2-20210512",
  "el8-ppc64le-appstream-r8.3-20210512",
  "el8-ppc64le-baseos-n8.4-20210512",
  "el8-ppc64le-baseos-n8.5-20210512",
  "el8-ppc64le-baseos-r8.2-20210512",
  "el8-ppc64le-baseos-r8.3-20210512",
  "el8-s390x-appstream-n8.4-20210512",
  "el8-s390x-appstream-n8.5-20210512",
  "el8-s390x-appstream-r8.2-20210512",
  "el8-s390x-appstream-r8.3-20210512",
  "el8-s390x-baseos-n8.4-20210512",
  "el8-s390x-baseos-n8.5-20210512",
  "el8-s390x-baseos-r8.2-20210512",
  "el8-s390x-baseos-r8.3-20210512",
  "el8-x86_64-appstream-n8.4-20210512",
  "el8-x86_64-appstream-n8.5-20210512",
  "el8-x86_64-appstream-r8.2-20210512",
  "el8-x86_64-appstream-r8.3-20210512",
  "el8-x86_64-baseos-n8.4-20210512",
  "el8-x86_64-baseos-n8.5-20210512",
  "el8-x86_64-baseos-r8.2-20210512",
  "el8-x86_64-baseos-r8.3-20210512",
  "el8-x86_64-google-cloud-sdk-20210512",
  "el8-x86_64-google-compute-engine-20210512",
  "el8-x86_64-rt-n8.4-20210512",
  "el8-x86_64-rt-n8.5-20210512",
  "el8-x86_64-rt-r8.2-20210512",
  "el8-x86_64-rt-r8.3-20210512",
  "el9-aarch64-appstream-n9.0-20210512",
  "el9-aarch64-baseos-n9.0-20210512",
  "el9-ppc64le-appstream-n9.0-20210512",
  "el9-ppc64le-baseos-n9.0-20210512",
  "el9-s390x-appstream-n9.0-20210512",
  "el9-s390x-baseos-n9.0-20210512",
  "el9-x86_64-appstream-n9.0-20210512",
  "el9-x86_64-baseos-n9.0-20210512",
  "el9-x86_64-rt-n9.0-20210512",
  "f33-aarch64-updates-released-20210512",
  "f33-aarch64-updates-released-modular-20210512",
  "f33-x86_64-updates-released-20210512",
  "f33-x86_64-updates-released-modular-20210512",
  "f34-aarch64-fedora-20210512",
  "f34-aarch64-fedora-modular-20210512",
  "f34-aarch64-updates-released-20210512",
  "f34-aarch64-updates-released-modular-20210512",
  "f34-x86_64-fedora-20210512",
  "f34-x86_64-fedora-modular-20210512",
  "f34-x86_64-updates-released-20210512",
  "f34-x86_64-updates-released-modular-20210512",
  "f35-x86_64-rawhide-20210512",
  "f35-x86_64-rawhide-modular-20210512"
]
```
